packages/salesforcedx-vscode-apex/package.json

```diff
@@ -23,7 +23,6 @@
   },
   "categories": ["Languages"],
   "devDependencies": {
-    "@salesforce/salesforcedx-utils-vscode": "40.4.0",
```

Comment:
@guw - Was there a reason you duplicated this in both devDependencies and dependencies?